ajax/libs/jquery-localize/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jquery-localize",
+  "filename": "jquery.localize.min.js",
+  "description": "A jQuery plugin that makes it easy to i18n your static web site.",
+  "author": "coderifous",
+  "license": "Apache-2.0",
```

Comment:
Let's use 
```js
"licenses": [
  "MIT",
  "GPL-2.0"
]
```
As README said https://github.com/coderifous/jquery-localize#credits--licensing
Thank you.

ref: https://www.openfoundry.org/enterprise-application/8680